pkg/alerts/alertsqlite/alerts_sqlite.go

```diff
@@ -625,49 +655,73 @@ func (p Sqlite) DeleteContactPoint(contact_id string) error {
 }
 
 // update last_sent_time and last_alert_state in notification_details table
-func (p Sqlite) UpdateLastSentTimeAndAlertState(alert_id string, alertState alertutils.AlertState) error {
+func updateLastSentTimeAndAlertState(tx *gorm.DB, alert_id string, alertState alertutils.AlertState) error {
```

Comment:
nit: I'm not quite sure what `tx` means (transaction?). Is this a really common name for a gorm DB? If not, a more descriptive name would be better, or perhaps just `db`

Reply:
Hmm, I am not sure what to actually name here, that is definitely a `db` instance from a Transaction. I think we need a name that represents that this is the `gorm.DB` instance from a Transaction. 
The only one I can think of `transactionDB`. Please let me know if you have any other suggestions.

Reply:
I'd say there's two options, depending on what information is important to the reader.
1. If for this particular function it doesn't matter that the DB is a DB from a transaction, then calling it `transactionDB` gives the reader information that isn't relevant, and—this probably doesn't matter for this specific case, but is helpful to think about in general—it could deter from other people reusing this function for a similar use case but with a non-transaction DB, because the name `transactionDB` makes them think the DB has to be from a transaction. So if this is the case, just call it `db` (or similar)
2. If it is important (just within the scope of this function) that this is a DB from a transaction, then I think `transactionDB` is a good name

I _think_ Option 1 is correct here, but I'm not certain

Reply:
Yes that make sense. As you have mentioned Option 1 is correct. If considered individually then that is just a `gorm.DB` instance. So I will just rename it to `db`.